spec/fuzzy-finder-spec.js

```diff
@@ -351,6 +360,44 @@ describe('FuzzyFinder', () => {
 
             expect(Array.from(projectView.element.querySelectorAll('li')).filter(a => a.textContent.includes('child-file.txt')).length).toBe(1)
           })
+
+          it('Return all the results if they have the same relative path across multiple root folders', async () => {
```

Comment:
I generally prefer test descriptions to read like an english sentence when appended to "it":

```
it("returns all the results if..."
```